client/router.js

```diff
@@ -34,6 +34,27 @@ FlowRouter.route('/legalnotice', {
 });
 
 
+FlowRouter.route( '/verify-email/:token', {
+    name: 'verify-email',
+    action( params ) {
+        Accounts.verifyEmail( params.token, ( error ) =>{
+            if ( error ) {
+                alert(error.reason);
+            } else {
+                FlowRouter.go( '/' );
+                alert('Email verified! Thanks!');
```

Comment:
Also: using alert() pop ups is hard to test in E2E headless browsers. Please avoid.
Flashmessages - as Felix pointed out look also a bit more beautiful. ;-)